src/Compute/Compute.Test/LiveTests/TestLiveScenarios.ps1

```diff
@@ -0,0 +1,33 @@
+Invoke-LiveTestScenario -Name "VM.NewVM Test" -Description "Test create new VM" -ScenarioScript `
+{
+    param ($rg)
+
+    $rgName = $rg.ResourceGroupName
+    $name = New-LiveTestResourceName
+
+    $vm = Get-AzVM -ResourceGroupName $rgName -Name $name 
+    if ($null -eq $vm) {
+        $actual = New-AzVM -Name $name -Credential (Get-Credential)
+    }
+    Assert-AreEqual $name $actual.Name
+
+    Assert-AreEqual $false $actual.EnabledForDeployment
+    Assert-Null $actual.Zone "By default Zone should be null"
+}
+
+Invoke-LiveTestScenario -Name "VM.RemoveVM Test" -Description "Test remove VM" -ResourceGroupLocation "eastus" -ScenarioScript `
```

Comment:
If it is not required to use a resource group location other than "westus", this optional parameter -ResourceGroupLocation is not necessary.